plugins/palo_alto_pan_os/bin/komand_palo_alto_pan_os

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Palo Alto Firewall"
 Vendor = "rapid7"
-Version = "6.1.3"
+Version = "6.1.4"
```

Comment:
You dont need to update the plugin version for updating the help.md. Can you please reuse 6.1.3

Reply:
We leave version, because we added information about connection to description of every action. If you would like we can of course reuse previous version.

Reply:
We dont want to release a new plugin for the help.md changes. If you use the existing version only help.md will be updated as part of the release. So, please use the existing production version of the plugin - 6.1.3.

Reply:
Since there are changes to schemas, the plugin version should be bumped.